Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeInstance.java

```diff
@@ -269,21 +269,21 @@ void setAccountId(Long accountId) {
      * @throws CentralRepoException if the Type's dbTableName has invalid
      *                        characters/format
      */
-    @Messages({"CorrelationType.FILES.displayName=Files",
-        "CorrelationType.DOMAIN.displayName=Domains",
-        "CorrelationType.EMAIL.displayName=Email Addresses",
-        "CorrelationType.PHONE.displayName=Phone Numbers",
-        "CorrelationType.USBID.displayName=USB Devices",
-        "CorrelationType.SSID.displayName=Wireless Networks",
-        "CorrelationType.MAC.displayName=MAC Addresses",
+    @Messages({"CorrelationType.FILES.displayName=File MD5",
+        "CorrelationType.DOMAIN.displayName=Domain",
+        "CorrelationType.EMAIL.displayName=Email Addresse",
+        "CorrelationType.PHONE.displayName=Phone Number",
+        "CorrelationType.USBID.displayName=USB Device",
+        "CorrelationType.SSID.displayName=Wireless Network",
+        "CorrelationType.MAC.displayName=MAC Addresse",
```

Comment:
Typo in Address here and above in "Email Adresse"

Reply:
Fixed the typos.